open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/relationships/SubjectAreaLine.java

```diff
@@ -59,14 +58,14 @@ public SubjectAreaLine(SubjectAreaRestClient subjectAreaRestClient, String... pa
                     ctor.setAccessible(true);
                     final AbstractSubjectAreaRelationship newInstance =
                             (AbstractSubjectAreaRelationship) ctor.newInstance(subjectAreaRestClient);
-                    cache.put(newInstance.type(), newInstance);
+                    cache.put(newInstance.resultType(), newInstance);
                 }
             } catch (NoSuchMethodException
                     | IllegalAccessException
                     | InstantiationException
                     | InvocationTargetException e) {
                 throw new ExceptionInInitializerError(
-                        "During initialization SubjectAreaLine an error has occurred - "
+                        "During initialization `SubjectAreaLine` an error has occurred - "
```

Comment:
please use the standard way of doing messages for subject area -add a new entry into SubjectAreaErrorCode and refer to that